### PR TITLE
Add recipe for netgen/content-browser

### PR DESCRIPTION
**Pull request by emodric**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: netgen/content-browser/0.12/config/routes/netgen_content_browser.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+netgen_content_browser:
+    resource: "@NetgenContentBrowserBundle/Resources/config/routing.yml"
+    prefix: "%netgen_content_browser.route_prefix%"
```
[the remark this answers is not in the document]
Comment (emodric) [reply netgen/content-browser/0.12/config/routes/netgen_content_browser.yaml]: Why is that? I don't want to use a hardcoded string. The parameter worked fine for the past three years.

**Comment by emodric** [comment]
The license is a business decision, which has already been made and cannot be changed, I'm afraid.

**Comment by fabpot** [comment]
Same for Symfony. We decided to not support not MIT/BSD licenses. Not something that is going to change. So, let's close here.

**Comment by emodric** [comment]
Sad day for Symfony community...

And I thought that diversity and inclusiveness is a thing...

**Comment by javiereguiluz** [comment]
@emodric just a quick comment: "diversity and inclusiveness" has nothing to do with people preferences about software licenses. It's better explained here: https://symfony.com/coc

**Comment by emodric** [comment]
@javiereguiluz It is very convenient when you decide what diversity is all about isn't it? So when someone challenges it, you have an exit clause. That's not how diversity works. In my opinion, someones choice of license is rooted in am opinion on how open source community should function. If this is not important to diversity initiative, than that's too sad.

**Comment by nicolas-grekas** [comment]
A license is not an opinion, it's a legal document that judges can enforce. You can have opinions about licenses but they are not opinions by themselves. That's why I agree with Javier here.

**Comment by Nyholm** [comment]
@emodric Please stop being passive aggressive. 
If Symfony made a decision what licenses and your company made a decision to go with anther license, then I guess we cannot play together with the recipes. Im sorry about that but that is appearance how it is. 

If this thread does not get any productive responses I will lock it.

**Comment by emodric** [comment]
A choice of license is an opinion, whether you like it or not. I'm willing to bet that most of developers choose their licenses based on their point of view and their opinion on how open source community should work.

@nyholm I'm not being passive aggresive. I'm trying to challenge you on your decision, which you do not want to listen. I'm just dissapointed and said because it feels I'm battling windmills here.

**Comment by Nyholm** [comment]
I assure you that collectively, people in this thread has discussed pros and cons with licenses over 100 times. I apologize if it feels like you are not being listen to. 

Your request is heard but Symfony will not accept any other licenses at the moment.

**Comment by emodric** [comment]
I still don't understand the decision to control the license on behalf of the developers. You are not the police and developers are not idiots. It's not as if you can block installing non-MIT code anyhow.

In my opinion, you're doing a disservice to the community, but if that is your decision, I'll leave it at that.

**Pull request closed without merging**